_docs/style/typography.md

```diff
@@ -16,170 +16,245 @@ helix-ui-javascript: false
+### Link
 
-*Do not subdue text on backgrounds darker than Gray 25 (#fafafa) or it will not pass accessibility standards.*
+Links support styles for hover, active, and disabled states, and inherit size and spacing styles from the text to which they are applied. Helix does not support the visited state.
 
 {% endcolumn %}
 
 {% column right:"hxCol-8 hxCol-xs-12 hxCol-sm-12 hxCol-md-8 hxCol-lg-8" %}
-{% include type-styles.html htmlexample='<p class="hxSubdued">Subdued Style</p>' typeface="Roboto" csscode="Gray 900" hex="#424242" %}
+{% figure [class:"image bg-light border"] %}
+![]({{site.url}}/assets/images/style/typography/type-link.png){:width="635px"}
+{% endfigure %}
 {% endcolumn %}
-
 </div>
 
-### Sub-body text
-
 <div class="hxRow" markdown="1">
 {% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
 
-Use sub-body text sparingly, since it is the least readable size. You can use it when smaller text is appropriate, such as for icon labels.
+### Caption
+
+Captions apply to images and visualizations. They should be used to clarify images and not introduce new, pertinent information.
 
 {% endcolumn %}
 
 {% column right:"hxCol-8 hxCol-xs-12 hxCol-sm-12 hxCol-md-8 hxCol-lg-8" %}
-{% include type-styles.html htmlexample='<p class="hxSubBody">Sub-Body Text</p>' typeface="Roboto" weight="Regular (400)" fontsize="12px / 0.75rem" csscode="Gray 900" hex="#424242" linespace="1.5" %}
+{% figure [class:"image bg-light border"] %}
+![]({{site.url}}/assets/images/style/typography/type-caption.png){:width="635px"}
+{% endfigure %}
 {% endcolumn %}
 </div>
 
-### Caption
+<div class="hxRow" markdown="1">
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+
+### Sub-body text
+
+Use sub-body text sparingly, since it is the least readable size. For example use this typography style for the following:
+
+- Labeling iconography
+- Labeling the legend on visualizations
+- Providing tertiary details or cards and tiles
+
+{% endcolumn %}
+
+{% column right:"hxCol-8 hxCol-xs-12 hxCol-sm-12 hxCol-md-8 hxCol-lg-8" %}
+{% figure [class:"image bg-light border"] %}
+![]({{site.url}}/assets/images/style/typography/type-sub-body.png){:width="635px"}
+{% endfigure %}
+{% endcolumn %}
+</div>
 
 <div class="hxRow" markdown="1">
 {% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
 
-Captions apply to images and visualizations. They should be used to clarify images and not introduce new, pertinent information.
+### Subdued style
+
+Use the subdued style to allow more important information stand out. For example:
+
+- Use the subdued style for keys in key-value lists to provide more visual prominenance to the value.
+- Use this style for non-critical information in lists, such as abbreviated dates, or as a second line of text in a cell
+
+Parameters for use: 
+
+- This style only applies Gray 750 color to the selected text, and does not affect font size or weight
```

Comment:
Remove the comma, and add a period at the end.

Reply:
updated.